packages/nl-design-system-unstable/__docs__/content-richtlijnen.md

```diff
@@ -22,42 +22,39 @@ Bv. help bestanden, veelgestelde vragen, contactgegevens.
 
 Bv. knoptekst, formulierlabels, feedbacktekst, contextuele helptekst.
 
-## Toegankelijkheid
+## Voor wie schrijven we
 
-Taal en toegankelijkheid gaan hand in hand. Als de tekstuele inhoud van een product niet of moeilijk begrepen wordt, dan is het product niet toegankelijk.
+Taal en toegankelijkheid gaan hand in hand. Als de tekstuele inhoud van een product niet of moeilijk begrepen wordt, dan is het product niet toegankelijk. Duidelijke taal zorgt er voor dat iedereen begrijpt wat je met je tekst wilt overbrengen.
```

Comment:
"tekstuele inhoud van een product" → "tekst over een product"?

Reply:
Rephrased this sentence.